File: C++/ВУЗ_06/ВУЗ_06/ВУЗ_06.cpp
// ВУЗ_05.cpp : Этот файл содержит функцию "main". Здесь начинается и заканчивается выполнение программы.
//

#include <iostream>

using namespace std;

int main() {
	setlocale(LC_ALL, "RUS");
	double a, b, c, d;
	cout << "Введите число a: " << endl;
	cin >> a;
	cout << "Введите число b: " << endl;
	cin >> b;
	cout << "Введите число c: " << endl;
	cin >> c;

	d = (b * b) - 4 * a * c;

	if (a == 0)
	{
		cout << "x: " << (-c / b) << endl;
	}
	else if (d > 0)
	{
		cout << "x1: " << (-b + sqrt(d)) / (2 * a) << endl;
		cout << "x2: " << (-b - sqrt(d)) / (2 * a) << endl;
	}
	else if (d == 0)
	{
		cout << "x: " << -b / (2 * a) << endl;
	}
	else
	{
		cout << ("Нет корней");
	}

	system("pause>nul");
	return 0;
}

File: Python/ВУЗ первый курс/ВУЗ_06.py
print("Введите число a: ");
a = float(input());
print("Введите число b: ");
b = float(input());
print("Введите число c: ");
c = float(input());

d = (b * b) - 4 * a * c;

if (a == 0):
        print("x: ", (-c / b));
elif (d > 0):
        print("x1: "(-b + sqrt(d)) / (2 * a));
        print("x2: "(-b - sqrt(d)) / (2 * a));
elif (d == 0):
        print("x: ", -b / (2 * a));
else:
        print("Нет корней");

